droidlet/tools/hitl/utils/allocate_instances.py

```diff
@@ -251,8 +315,14 @@ def free_ecs_instances(instance_ids):
     parser.add_argument(
         "--user", type=str, default="rebeccaqian@fb.com", help="Email of the CloudFlare account"
     )
+    parser.add_argument(
+        "--image_tag", type=str, help="The tag of docker image that will be used to spin up ecs instance"
+    )
+    parser.add_argument(
+        "--task_name", type=str, help="Task name of the ecs instance to be requested"
```

Comment:
Should task_name be an argument of allocate_instances on line 268 so that it can be passed to request_instance on line 269?

Reply:
you are right, I only tested the fn `request_instance()` separately. In fact there's some pieces missing, e.g. parameterizing image_tag & task_name in nsp_retrain main. Added those in the last two commits.